File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { SolardemoSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await SolardemoSDK.test()
    console.log('testsdk', testsdk)
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- Solardemo SDK exists test

local sdk = require("solardemo_sdk")

describe("SolardemoSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
